### ISSUE TITLE
[Errata] typo in practice exam ?

### ISSUE THREAD

**Issue by fugi303** [awsccp/awsccp.github.io#45]
Should the question be "performance **and** reliability" ? 
Otherwise I dont know what 'performance reliability' means.

![image](https://user-images.githubusercontent.com/116797194/198236990-e7c8e7da-ab17-48c7-8deb-56d3e76a42d0.png)

**Comment by dbclinton** [comment]
You're correct that the language of the question is a bit unclear. But the phrase "performance reliability" was intended to mean "reliability of performance." In other words, we're looking for tools that will make performance more predictable and reliable (and not necessarily that will *improve* performance).
Thanks for pointing this out.
